package.json

```diff
@@ -3,10 +3,21 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "prop-types": "^15.7.2",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "react-scripts": "3.4.3"
+    "react-scripts": "3.4.3",
+    "prop-types": "^15.7.2",
+    "@material-ui/core": "^4.11.0",
+    "@material-ui/icons": "^4.9.1",
+    "clsx": "^1.1.1",
```

Comment:
This package encourages very bad, non-uniform structure for class names. Use `classnames` instead: https://www.npmjs.com/package/classnames